Examples/Docs/Sources/AsyncExamples/main.swift

```diff
@@ -246,34 +247,94 @@ private func versionedAPI() throws {
     do {
         // Start Versioned API Example 2
 
-        // Use the `strict` option
-        var opts = MongoClientOptions()
-        opts.serverAPI = MongoServerAPI(version: .v1, strict: true)
+        let opts = MongoClientOptions(
+            serverAPI: MongoServerAPI(version: .v1, strict: true)
+        )
         let client = try MongoClient(uri, using: myEventLoopGroup, options: opts)
 
-        var findOpts = FindOptions()
-        findOpts.cursorType = .tailable
-        // Fails with an error because `tailable` is not part of version 1
-        let cursorFuture = client.db("db").collection("coll").find(options: findOpts)
-        cursorFuture.whenFailure { error in
-            // error
-            print(error)
-        }
-
         // End Versioned API Example 2
-        try? cursorFuture.wait()
+
         try client.syncClose()
     }
 
     do {
         // Start Versioned API Example 3
 
-        // Use the `deprecationErrors` option
-        var opts = MongoClientOptions()
-        opts.serverAPI = MongoServerAPI(version: .v1, deprecationErrors: true)
+        let opts = MongoClientOptions(
+            serverAPI: MongoServerAPI(version: .v1, strict: false)
+        )
+        let client = try MongoClient(uri, using: myEventLoopGroup, options: opts)
+
+        // End Versioned API Example 3
+
+        try client.syncClose()
+    }
+
+    do {
+        // Start Versioned API Example 4
+
+        let opts = MongoClientOptions(
+            serverAPI: MongoServerAPI(version: .v1, deprecationErrors: true)
+        )
         let client = try MongoClient(uri, using: myEventLoopGroup, options: opts)
+
         // End Versioned API Example 3
 
         try client.syncClose()
     }
 }
+
+private func versionedAPIMigrationExample() throws {
```

Comment:
this is a weird one... basically they want driver code equivalent to what is on this page: https://docs.mongodb.com/v5.0/reference/versioned-api/#example---migration